Gemfile

```diff
@@ -2,4 +2,8 @@
 # If you modify this file manually all comments and formatting will be lost.
 
 source "https://rubygems.org"
-gemspec :name => "logstash-core"
+gem "logstash-core", "2.0.0.dev", :path => "."
+gem "ci_reporter", "1.9.3", :group => :development
```

Comment:
the reason is that the testing dependencies do not belong in the `logstash-core` gem. The testing by Jenkins is done at the package level and so the testing dependencies belong in the top level package `Gemfile`. If you think about it, there is no relation with `ci_reporter` and the `logstash-core` gem, nothing in  `logstash-core` requires `ci_reporter`. The reason we never did that before is that it was not structured this way.

I do not think  `ci_reporter` belongs in `logstash-devutils` either, `ci_reporter` is really a package level tool used by Jenkins. Nothing in `logstash-devutils` depends on `ci_reporter`.

For the grouping, yes there is a problem. The `Gemfile` is managed by our `LogStash::Gemfile` class which does not (yet) support the `group` construct, it only support options hashes for the `gem` statements.